qase-javascript-commons/src/reporters/abstract-reporter.ts

```diff
@@ -1,20 +1,19 @@
 import { AxiosError } from 'axios';
 import get from 'lodash.get';
-
 import { TestResultType } from '../models';
-
 import { QaseError } from '../utils/qase-error';
 import { isAxiosError } from '../utils/is-axios-error';
+import { v4 as uuidv4 } from 'uuid';
 
 export interface LoggerInterface {
   log(message: string): void;
   group(): void;
   groupEnd(): void;
 }
 
-export type ReporterOptionsType = {
+export interface ReporterOptionsType {
   debug?: boolean | undefined;
-};
+}
```

Comment:
why no `;`?

Reply:
I have changed a type of object. ; - not required